server/integration_tests/test_data/e2e_edge_cases/povertyincaliforniaandcalifornia/chart_config.json

```diff
@@ -326,6 +326,399 @@
             "denom": "Count_Person",
             "title": "Population: Only English Spoken at Home, Below Poverty Line in Counties of California"
           },
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "statVarKey": [
+                      "Median_Income_Household"
+                    ],
+                    "title": "Household Median Income in California",
```

Comment:
From the SV list, what changed to cause this (and the addition in the next file)